clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -170,11 +183,14 @@ export class CopybookDownloadService implements vscode.Disposable {
         }
     }
 
-    private static async handleCopybook(dataset: string, cp: CopybookProfile, errors: Set<string>, isUSS: boolean) {
+    private static async handleCopybook(progress: vscode.Progress<{ message?: string; increment?: number }>, dataset: string, cp: CopybookProfile, errors: Set<string>, isUSS: boolean) {
         try {
             const fetchResult = await CopybookDownloadService.fetchCopybook(dataset, cp, isUSS);
             if (fetchResult) {
-                errors.delete(cp.copybook);
+                errors.delete(cp.getCopybook());
+                this.completedDownload++;
```

Comment:
Let's move this logic into the separate method as well.

Reply:
updated